test/grouping.jl

```diff
@@ -24,8 +25,47 @@ module TestGrouping
 
     byf = by(dt, :a, dt -> DataTable(bsum = sum(dt[:b])))
 
-    @test all(T -> T <: AbstractVector, map(typeof, colwise([sum], dt)))
-    @test all(T -> T <: AbstractVector, map(typeof, colwise(sum, dt)))
+    # colwise(::Vector{<:Function}, ::AbstractDataTable)
+    cw = colwise([sum], dt)
+    @test isa(cw, NullableArray{Any, 2})
+    @test size(cw) == (1,ncol(dt))
+    answer = NullableArray([20 12 -0.4283098098931877])
+    @test isequal(cw, answer)
+    nullfree = DataTable(Any[collect(1:10)], [:x1])
+    @test colwise([sum, minimum], nullfree) == reshape([55, 1], (2,1))
```

Comment:
Also test the type of the result like above? Same below for each case where you check the value.